src/main/java/org/medicmobile/webapp/mobile/EmbeddedBrowserActivity.java

```diff
@@ -330,6 +320,34 @@ public void locationRequestResolved() {
 		evaluateJavascript("window.CHTCore.AndroidApi.v1.locationPermissionRequestResolved();");
 	}
 
+//> PROTECTED
+	OnTouchListener onTouchEvent() {
```

Comment:
I think the idea makes sense. I'm just trying to fix an issue when doing the trick to display Settings Dialog, then press the phone's back button back to EmbeddedBrowserActivity. Then do the trick again but this time the Settings won't show, for some reason the listeners stop working. 
I have to navigate through tabs Messages to Reports, then the trick to open the Settings will work.

Reply:
Looks like the culprit is the OnKeyListener 🤔

Reply:
I changed the "secret door" to move forward since the OnKeyListener was giving me a lot of trouble with the back button and view scope. 
Now the user will have to tap 5 times with 1 finger anywhere on the screen, then swipe right with 2 fingers. Tap counting will restart if next tap happened more than 500 miliseconds since last tap.